app/views/categories/index.html.erb

```diff
@@ -0,0 +1,16 @@
+
+<h1 class="text-center">Categories</h1>
+
+<div id="categories">
+  <% @categories.each do |category| %>
+    <div class="card mb-3">
+      <div class="card-body">
+        <h5 class="card-title"><%= fa_icon "#{category.icon}" %><%= category.name %></h5>
+        <p class="card-text">Total amount of transactions: <%= category.deals.map {|deal| deal.amount }.sum %>$</p>
+        <%= link_to "Show this category", category, class: "btn btn-primary"%>
+      </div>
+    </div>
+  <% end %>
+</div>
+
+<%= link_to "New category", new_category_path, class: "category-button"%>
```

Comment:
- [ ] Kindly note that the design requirements say that you should follow these [design guidelines](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), including Colors, Typography: font face, size, and weight, Layout: composition and space between elements.
- Kindly make sure your category follows the template design layout

Your design | the template
---|---
![image](https://user-images.githubusercontent.com/65068771/176803441-a431f018-9b83-4be4-8f54-3944619d8517.png) | ![image](https://user-images.githubusercontent.com/65068771/176803510-b96e2a66-5545-4b39-825b-b91357109479.png)
- Make sure also a transaction category has an icon just like in the original template
- Note that also the new category button should always be at the bottom of the view(screen) even though there not a category in the database

Your design | the template 
---|---
![image](https://user-images.githubusercontent.com/65068771/176803874-9bcd1b17-4744-4d39-945a-67a4c18a854e.png) | ![image](https://user-images.githubusercontent.com/65068771/176803922-db394784-0731-4181-8e71-32ac18e67838.png)